natgeo/mortar
# [AEM-6430] Remove negative top margin from from POD heading at the mobile breakpoint

https://jira.natgeo.com/browse/AEM-6430

To test:

- Pull AEM-6430
- Run `npm test` to verify tests/linting pass
- Run `gulp serve`
- Verify mobile layout at http://localhost:3000/components/podpromo/
- From within mortar directory run `gulp build`
- From within the mortar directory, `cp lib/modules/podpromo/_styles.scss ~/Documents/workspace/ngs/aem-all/aem-platform/platform-view/node_modules/@natgeo/mortar_v3/lib/modules/podpromo/.` (note second path should match wherever you have your AEM code)
- From platform-view in AEM, `run mvn clean install -Pauto-deploy -s ../settings.xml`
- Create POD component and verify overlap is no longer occurring on at the mobile breakpoint
